src/Psalm/Storage/ClassConstantStorage.php

```diff
@@ -6,13 +6,32 @@
 use Psalm\Internal\Scanner\UnresolvedConstantComponent;
 use Psalm\Type;
 
+/**
+ * @psalm-suppress PossiblyUnusedProperty
```

Comment:
Shouldn't this be at the property level? Should I do a separate pull request for that, or is it intentional?